LICENSE-APACHE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2022-present Predrag Gruevski
```

Comment:
Technically, you aren't the sole copyright holder.

I've also heard that copyright date ranges are superfluous.

Reply:
>  if you'd like to be a co-copyright holder, happy to have you there. Yo

Unless there is explicit copyright re-assignment, all contributors hold the copyright on their contribution.

Reply:
Please do insert your name there and in the `README.md`, or let me know if you'd prefer `Ed Page` or something else and I can add it.

I once worked with a lawyer who felt the date range was important, so unless it hurts something I'm a bit inclined to keep it.

Reply:
> Please do insert your name there and in the README.md, or let me know if you'd prefer Ed Page or something else and I can add it.

Are you going to have every contributor edit the file to include their name?  I've not heard of a project doing that.

> I once worked with a lawyer who felt the date range was important, so unless it hurts something I'm a bit inclined to keep it.

I've heard from laywers the opposite :)

Reply:
I just took a peek at the cargo repo, and it seems that the `LICENSE-APACHE` file there just left the `Copyright [yyyy] [name of copyright owner]` line unchanged. We could do the same?

As another option, we could add text to the effect of "Copyright: contributors to this repo, as determined by git commit history" or some such thing.

I'm not a lawyer nor can I play one on TV, so I don't know what the right call is here. Since this code is hopefully going into cargo which is your domain, I'm happy to let you pick what you think is best.

Reply:
Deferring to git is a nice middle ground for now.

Technically, explicit copyright notices aren't needed due to the Berne Convention (e.g. some people put copyright headers in their code).   The main reason people do any of this is to be explicit about what is implicit and if any part gets pulled out of context, at least some will be left behind.

Reply:
Let me know what you think of the current wording. I don't have strong feelings about it, so if you have a preference for other wording, let me know or even just push to this branch directly — I don't mind.

Reply:
Good enough for me.